src/main/java/roomescape/RoomescapeAdminController.java

```diff
@@ -0,0 +1,20 @@
+package roomescape;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping("/admin")
+public class RoomescapeAdminController {
+
+    @GetMapping("")
```

Comment:
```suggestion
    @GetMapping
```
채울 내용이 없는 경우, 생략하는게 보기 편하기도 해요~